vsts/internal-version-increment.sh

```diff
@@ -75,8 +75,3 @@ fi
 if [ "$BUILD_SOURCEBRANCHNAME" != "master" ]; then
   sed -i '' 's/\(VERSION_STRING[[:space:]]*=[[:space:]]*\).*/\1'$publish_version'/g' $VERSION_FILENAME
 fi
-
-# Update swift package manager config file
-swift_package_file='Package.swift'
```

Comment:
It looks like the good place for it is `vsts/post-release-version-bump.sh`, isn't it?

Reply:
That would still require for manual check then

Reply:
Also, I can't find the pipeline using this script 🤔